### PR TITLE
Incremented version to 1.3.4.

### PR DESCRIPTION
**Pull request by zelinh**
Signed-off-by: Zelin Hao <zelinhao@amazon.com>

### Description
Increment to 1.3.4

### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/2205

Is this a backport? If so, please add backport PR # and/or commits #

### Testing
[Please provide details of testing done: unit testing, integration testing and manual testing]

### Check List
- [ ] New functionality includes testing
- [ ] New functionality has been documented
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peternied — APPROVED** [review]
Thanks Zelin.  A note on the WhiteSource failure, we need to coordinate the fix between OpenSearch and Security to get the right version in alignment.

```
Upgrade to version: com.fasterxml.jackson.core:jackson-databind:2.12.6.1,2.13.2.1
```

I'll reopen https://github.com/opensearch-project/security/issues/1687 to track this

**Comment by peternied** [comment]
@opensearch-project/security Could I get a second review on this change?

**Review by cliu123 — APPROVED** [review]

**Comment by cliu123** [comment]
Re-opened https://github.com/opensearch-project/security/issues/1687.

**Pull request merged**
